selfdrive/car/ford/interface.py

```diff
@@ -94,7 +100,7 @@ def _update(self, c):
     events = self.create_common_events(ret, extra_gears=[GearShifter.manumatic])
     if not self.CS.vehicle_sensors_valid:
       events.add(car.CarEvent.EventName.vehicleSensorsInvalid)
-    if self.CS.hybrid_platform:
+    if self.CS.hybrid_platform and self.CP.carFingerprint not in CANFD_CAR:
```

Comment:
can you add a short comment explaining this?

Reply:
we already have this in carstate.py, I meant that hybrid CANFD cars are not affected

Reply:
https://github.com/commaai/openpilot/pull/30709